src/main/java/com/vmware/vipclient/i18n/VIPCfg.java

```diff
@@ -393,4 +411,24 @@ public void setCachePath(String cachePath) {
         this.cachePath = cachePath;
     }
 
+	public String getOfflineResourcesBaseUrl() {
+		return offlineResourcesBaseUrl;
+	}
+
+	public void setOfflineResourcesBaseUrl(String offlineResourcesBaseUrl) {
+		this.offlineResourcesBaseUrl = offlineResourcesBaseUrl;
+	}
+
+	public List<DataSourceEnum> getMsgOriginsQueue() {
+		return msgOriginsQueue;
+	}
+
+	public void addMsgOriginsQueue(DataSourceEnum dataSource) {
+		this.msgOriginsQueue.add(dataSource);
+	}
+
+	public void setMsgOriginsQueue(List<DataSourceEnum> msgOriginsQueue) {
```

Comment:
Remove this 2 methods or make them private?
They don't need to exported.

Reply:
This is just a standard setter. This can be useful.